AntennaTracker/GCS_Mavlink.cpp

```diff
@@ -51,7 +51,8 @@ MAV_MODE GCS_MAVLINK_Tracker::base_mode() const
     }
 
     // we are armed if safety switch is not disarmed
-    if (hal.util->safety_switch_state() != AP_HAL::Util::SAFETY_DISARMED) {
+    if (hal.util->safety_switch_state() != AP_HAL::Util::SAFETY_DISARMED && tracker.control_mode != INITIALISING 
```

Comment:
This looks like a behaviour change; why take out the control-mode-initialising check?  Not saying it's wrong, but I believe it was there to avoid the GCS whinging about safety when the vehicle first boots...

Reply:
this just to make it report the same state that it is internally,  previously you could be disarmed and it would still report armed.